### PR TITLE
 2.4 AWS/GCP Upgrade Doc Updates

### PR DESCRIPTION
**Pull request by zkayyali812**
Ready for review

### CODE REVIEW

File: stories/assembly-aws-upgrade.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+[id="assembly-aws-application-upgrade"]
+
+:context: aws-application-upgrade
+
+= Upgrade
+
+You can upgrade your existing {PlatformNameShort} deployment to the newer version. 
+The upgrade process covers upgrading the {HubName}, {ControllerName}, and extension nodes.
+The upgrade process takes roughly the same amount of time as a {PlatformNameShort} deployment install. 
+It is required to create a backup before running the upgrade.
```
Comment (ianf77) [inline stories/assembly-aws-upgrade.adoc:12]: You are required

File: stories/assembly-gcp-upgrade.adoc [hunk 2]
```diff
@@ -4,41 +4,39 @@ ifdef::context[:parent-context: {context}]
 
 :context: gcp-upgrade
 
-= Upgrading your deployment
+= Upgrade
```
Comment (ianf77) [inline stories/assembly-gcp-upgrade.adoc:7]: Please leave procedure titles in the gerund form.

File: stories/assembly-gcp-upgrade.adoc [hunk 3]
```diff
@@ -4,41 +4,39 @@ ifdef::context[:parent-context: {context}]
...
-. Upgrade {PlatformNameShort}.
-.. Pull the `ansible-on-clouds-ops` 2.3 container image
-.. Generate data files by running the `ansible-on-clouds-ops` container
+. Upgrade {PlatformNameShort}
+.. Pull the ansible-on-clouds-ops 2.4 container image
+.. Ensure minimum required permissions are met
+.. Generate the data file
 .. Update the data file
-.. Run the `ansible-on-clouds-ops` 2.3 container to upgrade your {AAPonGCP} deployment
+.. Begin upgrading {PlatformNameShort}
```
Comment (ianf77) [inline stories/assembly-gcp-upgrade.adoc:36]: How?

File: stories/topics/proc-aws-generate-upgrade-data-file.adoc [hunk 4]
```diff
@@ -1,16 +1,16 @@
 [id="proc-aws-generate-upgrade-data-file_{context}"]
 
-= Generating the upgrade data file
+= Generate the data file
```
Comment (ianf77) [inline stories/topics/proc-aws-generate-upgrade-data-file.adoc:3]: Generating

File: stories/topics/proc-aws-upgrade-pull-container-image.adoc [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-= Pulling the ansible-on-clouds-ops 2.3 container image
+= Pulling the ansible-on-clouds-ops 2.4 container image
 
 .Procedure
 * Pull the Docker image for Ansible on Clouds operational container with the same tag as the version you want to upgrade to.
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com. 
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-aws-upgrade-pull-container-image.adoc:10]: ensure you are logged in

File: stories/topics/proc-gcp-generate-upgrade-data-file.adoc [hunk 6]
```diff
@@ -1,27 +1,32 @@
 [id="proc-gcp-generate-upgrade-data-file"]
 
-= Generating the data file
+= Generate the data file
```
Comment (ianf77) [inline stories/topics/proc-gcp-generate-upgrade-data-file.adoc:3]: Generating

File: stories/topics/proc-gcp-upgrade-pull-container-image.adoc [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
+= Pulling the ansible-on-clouds-ops 2.4 container image
 
 .Procedure
-* Pull the docker image for the `ansible-on-clouds-ops` 2.3 container with the same tag as the version you want to upgrade to.
+* Pull the Docker image for Ansible on Clouds operational container with the same tag as the version you want to upgrade to.
 +
 [NOTE]
 ====
-Before pulling the docker image, make sure you are logged in to registry.redhat.com using docker. Use the following command to login to registry.redhat.com.
+Before pulling the docker image, make sure you are logged in to registry.redhat.io using docker. Use the following command to login to registry.redhat.io.
```
Comment (ianf77) [inline stories/topics/proc-gcp-upgrade-pull-container-image.adoc:10]: ensure you are logged in

**Review by ianf77 — COMMENTED** [review]
Some comments

File: stories/topics/proc-aws-upgrade-pull-container-image.adoc (hunk 5 above)
Comment (zkayyali812) [reply stories/topics/proc-aws-upgrade-pull-container-image.adoc:10]: I will leave this change to be updated in this PR - https://github.com/ansible/aap-docs/pull/167 if that is ok

File: stories/topics/proc-gcp-upgrade-pull-container-image.adoc (hunk 7 above)
Comment (zkayyali812) [reply stories/topics/proc-gcp-upgrade-pull-container-image.adoc:10]: I will leave this change to be updated in this PR - https://github.com/ansible/aap-docs/pull/167 if that is ok

File: stories/topics/proc-aws-upgrade-pull-container-image.adoc (hunk 5 above)
Comment (zkayyali812) [reply stories/topics/proc-aws-upgrade-pull-container-image.adoc:10]: I will revert the earlier change to allow it to be handled in #167

File: stories/topics/proc-gcp-upgrade-pull-container-image.adoc (hunk 7 above)
Comment (zkayyali812) [reply stories/topics/proc-gcp-upgrade-pull-container-image.adoc:10]: I will revert the earlier change to allow it to be handled in https://github.com/ansible/aap-docs/pull/167

**Comment by zkayyali812** [comment]
@ianf77 Thank you for the detailed reviews. Updated with all relevant notes!
If all changes look good to you, this PR can be merged

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**